bplist/bplist.py

```diff
@@ -62,10 +62,18 @@ def __init__(self, s):
         self.data = s
         self.objects = []
         self.resolved = {}
-    
+
+    def __getObjHeader(self, offset):
+        obj_header = self.data[offset]
+
+        if type(obj_header) == str:
```

Comment:
I'm pretty sure we don't need that check because the BPlistReader() expects an `str` as input.

Reply:
So reading from string never worked or something changed in python2?

Reply:
It's a tricky question to answer now, given I never wrote unit tests for this :-)

I'm pretty sure all codepaths used to work, but I went back to python 2.5 and it's still a problem there. This package is clearly not the most popular one so maybe it was a bug all these years.

Reply:
In that case, I feel like it's safer to leave the check in there